packages/flutter/lib/src/widgets/animated_cross_fade.dart

```diff
@@ -216,7 +216,6 @@ class AnimatedCrossFade extends StatefulWidget {
   /// [AnimatedCrossFadeBuilder].
   static Widget defaultLayoutBuilder(Widget topChild, Key topChildKey, Widget bottomChild, Key bottomChildKey) {
     return Stack(
-      overflow: Overflow.visible,
```

Comment:
This will have to change back to `clipBeahvior: none`, no?

Reply:
Yes!